requirements.txt

```diff
@@ -59,3 +59,7 @@ requests==2.20.0
 
 # sentry
 sentry-sdk==0.14.1
+
+# slugify
+python-slugify==4.0.0 
```

Comment:
We should already have a method of creating slugs in the project, let's use that instead of adding a new dependency.